attributes/default.rb

```diff
@@ -131,9 +131,17 @@
 default['cluster']['slurm']['user_id'] = node['cluster']['reserved_base_uid'] + 1
 default['cluster']['slurm']['group'] = node['cluster']['slurm']['user']
 default['cluster']['slurm']['group_id'] = node['cluster']['slurm']['user_id']
+default['cluster']['slurm']['dbduser'] = 'slurmdbd'
+default['cluster']['slurm']['dbduser_id'] = node['cluster']['reserved_base_uid'] + 5
```

Comment:
[Question] Would it make sense to have the uid offsets centralized somewhere to ensure we don't accidentally create a collision or is this not something to worry about?

Reply:
Absolutely legit question. It looks like the centralized place is the `default.rb` file itself: the offsets are just scattered around the file because they are defined where they are used.

I guess it makes more sense to keep them as they are because this way the users are defined in parts of the file that contain all the details related to why that user has been defined. And we can always just grep `reserved_base_uid` if we need all the offsets.

Reply:
We wiil add it to the extra tasks for this feature.